.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "client/packages/plugins/stPlugins"]
```

Comment:
Do we want to check this in?

Reply:
Thanks: c4a504cfc538e8b292556749f7362dcbadc70496